pkg/coscheduling/core/core.go

```diff
@@ -155,15 +163,18 @@ func (pgMgr *PodGroupManager) PreFilter(ctx context.Context, pod *corev1.Pod) er
 	if _, ok := pgMgr.lastDeniedPG.Get(pgFullName); ok {
 		return fmt.Errorf("pod with pgName: %v last failed in %v, deny", pgFullName, *pgMgr.lastDeniedPGExpirationTime)
 	}
-	pods, err := pgMgr.podLister.Pods(pod.Namespace).List(
-		labels.SelectorFromSet(labels.Set{v1alpha1.PodGroupLabel: util.GetPodGroupLabel(pod)}),
-	)
-	if err != nil {
-		return fmt.Errorf("podLister list pods failed: %v", err)
-	}
-	if len(pods) < int(pg.Spec.MinMember) {
-		return fmt.Errorf("pre-filter pod %v cannot find enough sibling pods, "+
-			"current pods number: %v, minMember of group: %v", pod.Name, len(pods), pg.Spec.MinMember)
+
+	if time.Now().After(pod.CreationTimestamp.Time.Add(*pgMgr.ignorePodNumCheckingTime)) {
```

Comment:
This time window would bypass the quorum checking, so I'm wondering if you can show some specific cases/data proving it's beneficial. I suppose the logic would not even be triggered in a busy cluster.

Reply:
in other words, without this window, reources won't be reserved until minMember is reached, otherwise the internal "denyGroup" works as a backoff timer; with this window, it might be possible some pods may _unconditionally_ reserve resources (if they're within that window). So I want to evaluate the pros & cons carefully.

Reply:
sorry for the delay, maybe the issue https://github.com/kubernetes-sigs/scheduler-plugins/issues/364 could be the case of quickly passing quorum checking (in this case, `MinResources` is set as `nil`).

For the resources reserving, I will test it carefully & maybe update code.